ui_framework/doc_site/src/views/side_nav/side_nav.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {
+  KuiSideNav,
+  KuiSideNavItem,
+  KuiSideNavTitle,
+} from '../../../../components';
+
+export default () => (
+  <KuiSideNav>
+    <KuiSideNavTitle>Elasticsearch</KuiSideNavTitle>
+    <KuiSideNavItem>Data sources</KuiSideNavItem>
+    <KuiSideNavItem>Users</KuiSideNavItem>
+    <KuiSideNavItem>Roles</KuiSideNavItem>
+    <KuiSideNavItem isSelected>Watches</KuiSideNavItem>
+    <KuiSideNavItem>Extremely long item is long</KuiSideNavItem>
```

Comment:
Can we make this example much longer, so we can see how content wraps? Maybe it should truncate with an ellipsis instead?